### PR TITLE
Create nx_version.txt

### PR DESCRIPTION
**Pull request by devotuoma**
Am happy to learn how to create PR but I would love to gain more light on Git and Github workflow

### CODE REVIEW

File: 2023-round-1/devotuoma/nx_version.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+2.7.1
+
```
Comment (MridulS) [inline 2023-round-1/devotuoma/nx_version.txt:2]: thanks for this @devotuoma!

Could you try upgrading your python version and networkx version?

**Comment by devotuoma** [comment]
Thank you I am updating them today and I will give the feedback

**Comment by devotuoma** [comment]
fine

**Comment by devotuoma** [comment]
I have created Networkx tutorial and python script.

File: 2023-round-1/devotuoma/nx_tutorial_script.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+#printing the source and destination of each possible paths 
+for s in shortest_path:
+    print("shortest paths from", s, ":")
+    for d in shortest_path[s]:
+        print( "to", d,":", shortest_path[s][d])
+
+colors = nx.get_node_attributes(G,'color').values() #getting the color attributes of each node
+
+nx.draw_networkx(G,with_labels=True,node_color=colors) #drawing the graph
```
Comment (MridulS) [inline 2023-round-1/devotuoma/nx_tutorial_script.py:27]: Thanks @devotuoma for the pull request.
This looks very similar to another script https://github.com/networkx/outreachy/blob/main/2023-round-1/Qudirah/nx_tutorial_script.py 

Could you rewrite the script from scratch? You can use the networkx tutorial https://networkx.org/documentation/latest/tutorial.html to learn more about how to create graphs and run algorithms. Thanks!

Comment (devotuoma) [reply 2023-round-1/devotuoma/nx_tutorial_script.py:27]: Am going to work on this.

**Comment by MridulS** [comment]
Thanks @devotuoma!

**Pull request merged**